src/widget/wnumberpos.h

```diff
@@ -20,26 +20,17 @@ class WNumberPos : public WNumber {
 
   private slots:
     void setValue(double dValue) override;
-    void slotSetPosition(double);
-    void slotSetTrackSampleRate(double dSampleRate);
-    void slotSetTrackSamples(double dSamples);
+    void slotSetTimeElapsed(double);
     void slotSetDisplayMode(double);
 
   private:
 
     TrackTime::DisplayMode m_displayMode;
 
-    // Old value set
-    double m_dOldPosition;
-    double m_dTrackSamples;
-    double m_dTrackSampleRate;
-    // True if remaining content is being shown
-    bool m_bRemain;
+    double m_dOldTime;
```

Comment:
For consistency and disambiguation this member should be named m_dOldTimeElapsed explicitly